presto-main/src/main/java/com/facebook/presto/operator/aggregation/AggregationCompiler.java

```diff
@@ -148,21 +128,14 @@ public InternalAggregationFunction generateAggregationFunction(Class<?> clazz, T
         return builder.build();
     }
 
-    private List<ParameterMetadata> getParameterMetadata(@Nullable Method method, boolean sampleWeightAllowed)
+    private static boolean isParameterNullable(Annotation[] annotations)
     {
-        if (method == null) {
-            return null;
-        }
-
-        ImmutableList.Builder<ParameterMetadata> builder = ImmutableList.builder();
-        builder.add(new ParameterMetadata(STATE));
+        return Arrays.asList(annotations).stream().anyMatch(annotation -> annotation instanceof NullablePosition);
```

Comment:
you can use: 
 - https://docs.oracle.com/javase/8/docs/api/java/util/stream/Stream.html#of-T...-
 - https://docs.oracle.com/javase/8/docs/api/java/util/Arrays.html#stream-T:A-
Instead of `Arrays.asList(...).stream()`